simplefs/simplefs.go

```diff
@@ -147,33 +147,37 @@ func remoteTlfAndPath(path keybase1.Path) (
 }
 
 func (k *SimpleFS) getFS(ctx context.Context, path keybase1.Path) (
-	fs billy.Filesystem, finalElem string, err error) {
+	fs billy.Filesystem, finalElem string, exitEarly bool, err error) {
```

Comment:
Instead of this new boolean flag, why not just log and reset the error?

Reply:
Caller would otherwise use the returned `fs` to `Stat` and `Readdir`, which in this case would be a `nil`. I guess we could just return early in caller if `fs == nil`, but I feel having a flag is better since it explicitly says the caller should exit early.

Reply:
I'd rather have a special error type to check against.